plotly/tests/test_core/test_tools/test_Quiver.py

```diff
@@ -0,0 +1,223 @@
+import plotly
+from plotly.graph_objs import *
+import plotly.tools as tls
+from nose.tools import raises
+import numpy as np
+
+
+@raises(Exception)
+def unequal_xy_length():
```

Comment:
does this test run? I think that you need to have `test` in the definition for it to register? might be wrong about that though.